torch/fx/experimental/symbolic_shapes.py

```diff
@@ -3842,6 +3842,10 @@ def _update_var_to_range(self, symbol, vr):
         else:
             self.var_to_range[symbol] &= ValueRanges(lower, upper)
 
+        # If the range is refined to singleton, set replacement
```

Comment:
my point is that you need to then query again `self.var_to_range[symbol]` to get the updated lower / upper bound...

Reply:
> my point is that you need to then query again `self.var_to_range[symbol]` to get the updated lower / upper bound...

Oh I got you, so sorry for my misunderstanding